google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/ExternalTableDefinition.java

```diff
@@ -186,6 +186,12 @@ public Builder setObjectMetadata(String objectMetadata) {
 
     abstract Builder setObjectMetadataInner(String objectMetadata);
 
+    public Builder setMetadataCacheMode(String metadataCacheMode) {
```

Comment:
It might be nice to include the documented values in the docstring here and/or in the getter e.g AUTOMATIC, MANUAL, etc

Reply:
Updated. PTAL.